compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -964,3 +983,41 @@ impl<'a, 'tcx> MutVisitor<'tcx> for Integrator<'a, 'tcx> {
         }
     }
 }
+
+crate fn mir_inliner_callers_count_map<'tcx>(
+    tcx: TyCtxt<'tcx>,
+    _: (),
+) -> Lrc<FxHashMap<DefId, usize>> {
+    let mut callers_count_map: FxHashMap<DefId, usize> = Default::default();
+
+    // Iterate over all body owners of current crate
+    for did in tcx
+        .hir()
+        .body_owners()
```

Comment:
Should we use `mir_keys` here?

Reply:
`mir_keys` adds introduces for type check.